build-ig.md

```diff
@@ -0,0 +1,18 @@
+prior configure: you need pretty modern automake autoconf and libtoolize
```

Comment:
I don't think we want this file to be here.

Reply:
this is just build instruction on how to build this.
i dont see anything wrong with adding this.